lib/SPIRV/libSPIRV/SPIRV.debug.h

```diff
@@ -1021,10 +1021,10 @@ inline bool hasDbgInstParentScopeIdx(
     ParentScopeIdx = LocalVariable::ParentIdx;
     return true;
   case SPIRVDebug::ImportedEntity:
-    if (ExtKind == SPIRV::SPIRVEIS_NonSemantic_Shader_DebugInfo_200)
-      ParentScopeIdx = ImportedEntity::NonSemantic::ParentIdx;
-    else
+    if (ExtKind == SPIRV::SPIRVEIS_OpenCL_DebugInfo_100)
       ParentScopeIdx = ImportedEntity::OpenCL::ParentIdx;
+    else
+      ParentScopeIdx = ImportedEntity::NonSemantic::ParentIdx;
```

Comment:
I guess it's impossible case that debug info is generated with other spec (not OpenCL and not NonSemantic)?